data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+
+mission "Coalition: Expeditions Past 3"
+	landing
+	name "Dusted Dead Drop"
+	description "Head to <destination>, where Sedlitaris wishes to check an ancient Saryd vault for the route and logs of her ancestor's expedition."
+	passengers 1
+	to offer
+		has "Coalition: Expeditions Past 2: done"
+	source "Ablub's Invention"
+	destination "Sundrinker"
+	on offer
+		conversation
+			`The Heliarch council representatives guide you far from the planet's equator, to a region close to a snowy mountain and tell you to land on top of a cube-shaped plateau. Dozens of searchlights are dotted along the edges, lighting the ground below, while a warehouse complex sits at the center of the plateau top. As you're approaching the plateau, your ship's sensors alert you to incoming vessels; you look at your radar to see a Saryd Sojourner, followed by three Travelers, getting closer to your ship. They appear unarmed, but as you land on the plateau and they follow you down, defensive turrets on the edges of the complex remain locked onto them.`
+			`	Once the Sojourner lands, a Saryd comes down the ramp and heads to your ship. Sedlitaris sighs and leaves to meet him, followed by the representatives of the Heliarch council. Once you're all out of your ship, you see the Saryd has been stopped by a trio of Arach guards, clad in winter uniforms and blocking the his path. "Told mother or father about this, have you?" he shouts at Sedlitaris as she walks closer. "Because important enough to be told, I apparently am not. That out to die you were, told I was by some Kimek arbiter that met, I never had!"`
+			`	"Repilor, a work trip, this is. Away for long, I won't be, so wish to trouble you all with the news, I didn't," she responds.`
+			`	"Trouble us? Truth is, know that wrong, you are, but face telling father that you're throwing your life away, you couldn't!"`
+			`	He tries to push his way to her, but the guards stop him before he can make much progress. One of the Heliarch representatives gets closer, saying, "A restricted area, this is. Invading government property, you are, so suggest that you leave immediately, I do."`
+			`	"Restricted? What about the outside? Is it not also restricted? Much more dangerous? Fine with sending my sister there, you seem," Repilor counters, and the Heliarch looks away after a moment. "If bothering the consuls so much, she was, just fired, she should've been!" He then looks at you, and continues, "And the human? Your transport out of here, I imagine? Or maybe the one who put these ideas in your head, they are?"`
+			choice
+				`	"I've just been hired to transport her."`
+					goto hired
+				`	"What's so wrong about her wanting to explore outside the Coalition?"`
+			`	"Awaiting 'explorers,' nothing but an unknown, certain doom is. Wish for that to be my sister's fate, I do not," he answers.`
```

Comment:
In this he's responding to the player's prompt about "what's wrong with wanting to **explore**" so I felt it'd be appropriate to have him use a word closer to that (explorers) in his reply.
If you think that gets inconsistent because of the rank name, could change it so that he doesn't have the quoted portion within his reply.

Reply:
Makes sense, the use of single quotes is just what threw me off. With the change of `Inspector` to a proper noun (see above unresolved conversation on line 2412), this should be fine.